timetable/XML/XmlCreator.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Xml;
+using System.Xml.Linq;
+
+
+namespace Timetable.timetable.XML
+{
+	public class XmlCreator
+	{
+		private XDocument xDocument { get; set; }
+		private static XmlCreator instance;
+		private string pathName { get; set; }
+        
+
+		/// <summary>
+		/// Initializes a new instance of the <see cref="T:Timetable.timetable.XML.XmlCreator"/> class.
+		/// </summary>
+		private XmlCreator()
+		{
+			DateTime localDate = DateTime.Now;
+			pathName = "FET Files/Timetable:" + localDate + ".fet";
```

Comment:
This throws the following exception for me:

```
System.NotSupportedException
  HResult=0x80131515
  Message=De opgegeven padindeling wordt niet ondersteund.
  Source=mscorlib
  StackTrace:
   at System.Security.Permissions.FileIOPermission.EmulateFileIOPermissionChecks(String fullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, Boolean useAsync)
   at System.Xml.XmlWriterSettings.CreateWriter(String outputFileName)
   at System.Xml.XmlWriter.Create(String outputFileName, XmlWriterSettings settings)
   at System.Xml.Linq.XDocument.Save(String fileName, SaveOptions options)
   at System.Xml.Linq.XDocument.Save(String fileName)
   at Timetable.timetable.XML.XmlCreator.Save() in [..]\timetable\XML\XmlCreator.cs:line 54
   at Timetable.Application.Main() in [..]\timetable\Application\Main.cs:line 50

```

Reply:
hmmm.. it is probably the filepath. I will fix this.